src/components/Welcome.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import getToken from '../lib/tokens';
+
+const Welcome = () => {
+  const setTokenStorage = () => {
+    const token = getToken();
+    localStorage.setItem('token', token);
+  };
+  return (
+    <div>
+      <h1>Welcome to your Smart Shopping list!</h1>
+      <Link exact to="/list" onClick={setTokenStorage}>
```

Comment:
It appears the current _"exact"_ keyword in the `<Link />` tag is causing the following console error: _"Received `true` for a non-boolean attribute `exact`."_ I believe it's safe to remove, though updating to `exact="true"` should also clean up the error, if preferred.